### PR TITLE
Add link to summary as PR comment

### LINKED ISSUE #403: Link to built documentation artifact download in PR comment
We build the docu in PRs.

We could maybe automatically add a comment to the PR that contains the link to the built documentation and update the link an rebuilds.

Not sure if there exists a github action for that already

### PR DESCRIPTION
**Pull request by ckolbPTB**

### CODE REVIEW

**Comment by fzimmermann89** [comment]
👍

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -192,11 +195,29 @@ jobs:
...
         with:
           name: Documentation
           path: docs/build/html/
 
+      - run: echo '${{ steps.save_docu.outputs.artifact-url }}'
+
+      - run: echo '${{ github.event.number }}'
+
+      - name: Update PR with link to summary
+        uses: edumserrano/find-create-or-update-comment@v2
```
Comment (fzimmermann89) [inline .github/workflows/docs.yml]: consider using v3 here to avoid node deprcation warning

**Comment by fzimmermann89** [comment]
consider also adding a link to this:
https://zimf.de/zipserve/https://github.com/PTB-MR/mrpro/actions/runs/11109144782/artifacts/1996243109/ as view online
so just but https://zimf.de/zipserve/ in front of the artifact url and a slash at the end

**Comment by ckolbPTB** [comment]
> consider also adding a link to this: https://zimf.de/zipserve/https://github.com/PTB-MR/mrpro/actions/runs/11109144782/artifacts/1996243109/ as view online so just but https://zimf.de/zipserve/ in front of the artifact url and a slash at the end

happy to but this binds you to MRpro for eternity - are you sure you are ready for this...

**Comment by fzimmermann89** [comment]
> > consider also adding a link to this: https://zimf.de/zipserve/https://github.com/PTB-MR/mrpro/actions/runs/11109144782/artifacts/1996243109/ as view online so just but https://zimf.de/zipserve/ in front of the artifact url and a slash at the end
> 
> happy to but this binds you to MRpro for eternity - are you sure you are ready for this...

The artifacts are only available for 90days anyways , and the links will stop being useful afterwards. So I am only bound for 90days :)

**Review by fzimmermann89 — APPROVED** [review]
i changed the text with some nice emojis, feel free to revert and merge the prev version or this one.

**Pull request merged**